packages/components/credentials/GoogleSearchApi.credential.ts

```diff
@@ -0,0 +1,29 @@
+import { INodeParams, INodeCredential } from '../src/Interface'
+
+class GoogleSearchApi implements INodeCredential {
+    label: string
+    name: string
+    version: number
+    description: string
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Google Custom Search API'
+        this.name = 'googleCustomSearchApi'
+        this.version = 1.0
```

Comment:
can we add a description like [this](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/credentials/ZepMemoryApi.credential.ts#L14) to let user know how and where to get these credentials?

Reply:
Sure. Just added the description.